1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts

```diff
@@ -5,8 +5,16 @@ describe('palindrome checker', () => {
     expect(palindromeChecker).toBeDefined();
   });
 
-  it('should know that mom is a palindrome', () => {
-    expect(palindromeChecker('mom')).toBeTruthy();
+  it.each([
+    'mom',
+    'Mom',
+    'Was It A Rat I Saw',
+    'MoM',
+    'xMomx',
+    'Never Odd or Even',
+    '1Never Odd or Even1'
+  ])('when given %s should return true', (param: string) => {
```

Comment:
I'd name these parameterized test something like:
`('knows that %s is a palindrome', (param: string) => {`

Reply:
Ah yes, great spot! I will find that lesson again and re-read!